File: OpenMP/sharedhello.c
#include <stdio.h>
#include  "omp.h"
int main(void)
{
	char greetings[] = "Hello world"; 
	printf("Before parallel section %s\n", greetings); 

	#pragma omp parallel num_threads(4) private(greetings)
	{
	char greetings[] = "Saluton mondo"; 
	printf("Inside parallel section %s\n", greetings); 
	}
	printf("After parallel section %s\n", greetings); 
return 0;
}

File: OpenMP/sharedhello.f90
program SharedHello
implicit none
        character(len=16) :: greetings 
        greetings = "Hello World!" 
        print *, "Before parallel section: ", greetings 

        !$omp parallel num_threads(4) private(greetings)
        greetings = "Saluton mondo!"
        print *, "Inside parallel section: ", greetings
        !$omp end parallel

        print *, "After parallel section:  ", greetings

end program SharedHello
